### PR TITLE
Add Dari (prs) and Pashto (ps) translations.

### PR DESCRIPTION
**Pull request by gimite**

### CODE REVIEW

File: app/locale/ps/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -0,0 +1,1379 @@
...
+msgstr ""
+
+msgid "(click to reveal)"
+msgstr "(د سپړلو لپاره کليک کړئ)"
+
+msgid "(unknown)"
+msgstr "(نامعلوم)"
+
+msgid "A message for this person or others seeking this person"
+msgstr "د دې شخص يا نورو لپاره چې په لټه کې يې دي پيغام"
```
Comment (skywhale) [inline app/locale/ps/LC_MESSAGES/django.po]: why are some messages already translated?

**Comment by skywhale** [comment]
LGTM

**Comment by gimite** [comment]
Thanks! Merged.

File: app/locale/ps/LC_MESSAGES/django.po (hunk 1 above)
Comment (gimite) [inline app/locale/ps/LC_MESSAGES/django.po]: We asked volunteers to translate them. There are too many, so they chose only important messages.

**Pull request merged**